pajbot/modules/pnsl.py

```diff
@@ -15,6 +15,7 @@ class PNSLModule(BaseModule):
     NAME = "Run P&SL lists"
     DESCRIPTION = "Run P&SL lists through the !runpnsl command"
     CATEGORY = "Moderation"
+    CONFIGURE_LEVEL = 1500
```

Comment:
I personally think this module should be 1000, people who utilize this module are most likely 1000 or 2000, and almost never 1500.